src/Effect/Ref.js

```diff
@@ -1,26 +1,26 @@
 "use strict";
 
-exports.new = function (val) {
+export var _new = function (val) {
```

Comment:
I thought we'd settled on `export const ...`; is this simply an old commit that should be updated, or have we switched to `var`?

Reply:
At first, I was just opening up the PRs that were being used by the ES modules PR without any additional modifications. Then, I got done with 'core' libraries and worked on repos that didn't have that work already done. So, I started using `lebab` and some replacement stuff.

So, we still are going to switch  to `export const`. This PR just needs to be updated to have that.